### PR TITLE
Rapid prototyping bqml automl

### PR DESCRIPTION
**Pull request by SamyuktaDR**
**REQUIRED:** Add a summary of your PR here, typically including why the change is needed and what was changed. Include any design alternatives for discussion purposes.

<br>
Deleting Bigquery dataset using bigquery client was raising an issue. So instead replace the delete using inline bq command line function. Also added UUID to bq dataset name while creation.
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

**Comment by ivanmkc** [comment]
Missing a PR summary.

File: notebooks/official/structured_data/rapid_prototyping_bqml_automl.ipynb [hunk 1]
```diff
@@ -73,6 +73,27 @@
         "id": "k-82uiXlTjvw"
```
Comment (ivanmkc) [inline notebooks/official/structured_data/rapid_prototyping_bqml_automl.ipynb:73]: <pre class="ql-syntax" spellcheck="false">Line #2.    BUCKET_URI = f"gs://{BUCKET_NAME}"
</pre><p>Remove this line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/877/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (ivanmkc) [inline notebooks/official/structured_data/rapid_prototyping_bqml_automl.ipynb:73]: <pre class="ql-syntax" spellcheck="false">Line #3.        BUCKET_URI = f"gs://{BUCKET_NAME}"
</pre><p>Move this line outside the if statement.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/877/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (ivanmkc) [inline notebooks/official/structured_data/rapid_prototyping_bqml_automl.ipynb:73]: <pre class="ql-syntax" spellcheck="false">Line #1.    if os.getenv("IS_TESTING"):
</pre><p>Are lines 1 to 3 needed? Remove it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/877/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Comment (ivanmkc) [inline notebooks/official/structured_data/rapid_prototyping_bqml_automl.ipynb:73]: <p>Don't use first-person. Should be <code>you validate the endpoint by making a simple prediction to it.</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/877/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Comment (ivanmkc) [inline notebooks/official/structured_data/rapid_prototyping_bqml_automl.ipynb:73]: <pre class="ql-syntax" spellcheck="false">Line #41.    pipelines = vertex.PipelineJob.list(
</pre><p>Never get the pipeline by pipelines[-1]. It is flaky and non-deterministic. </p><p> </p><p>Get pipeline by a unique ID.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/877/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

**Review by ivanmkc — CHANGES_REQUESTED** [review]
Needs changes.

**Comment by SamyuktaDR** [comment]
PR Summary : Deleting Bigquery dataset using bigquery client was raising an issue. So instead replace the delete using inline bq command line function. Also added UUID to bq dataset name while creation.

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**